docs/cow-protocol/reference/core/auctions/ebbo_specifics.md

```
@@ -15,16 +14,14 @@ A certificate for an EBBO violation can be challenged by the solver who is accus
-The following steps are taken, in case an EBBO violation is detected, either by the core team’s monitoring infrastructure or a third party reach out (for example by the user themselves or by solvers). We clarify here that in order to avoid reports of very old trades that occurred at a time where the protocol was in earlier phases, violations will be inspected only if they are communicated within 3 months from the incident date.
-
-1. A violation is detected, within 3 months from the incident date.
+The following steps are taken, in case an EBBO violation is detected, either by the core team’s monitoring infrastructure or a third party reaching out (for example by the user themselves or by solvers). Violations will be inspected only if they are communicated within 3 months from the incident date.
 
-2. The core team calculates the (minimum) value lost to users in a given settlement due to the EBBO violation, as described in the section above.
+1. The core team calculates the value lost to users in a given settlement due to the EBBO violation, as described in the section above.
 
-3. The core team requests the involved solver team to reimburse the user directly, by informing them about the amount:
-  - The reimbursement is to be done in the surplus token and sent from a responsible solver’s submission / owned address for clarity about who issued the refund. If the solver has no access to their submission account (in case this is managed by the core team), then the core team should transfer any desired amount from the submission account to a solver-related address (i.e. their rewards account), if such a request is made from the solver.
+2. The core team requests the involved solver team to reimburse the user directly, by informing them about the amount.
+  - The reimbursement is to be done in the surplus token and sent from an address directly linked to the responsible solver for clarity about who issued the refund. If the solver has no access to their submission account (in case this is managed by the core team), the reimbursement can be executed from the rewards address.
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)